### PR TITLE
KEYCLOAK-16072 add mising Advanced Login Configuration documentation

### PR DESCRIPTION
**Pull request by pA1nD**
Add missing sections in docs for `Keycloak.prototype.redirectToLogin`

### CODE REVIEW

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 1]
```diff
@@ -266,6 +266,22 @@ for each section:
     app.get( '/:section/:page', keycloak.protect( protectBySection ), sectionHandler );
 ----
 
+Advanced Login Configuration:
+
+By default, all unauthorized requests will be redirected to the Keycloak login page unless your client is bearer-only.
```
Comment (andymunro) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]: Change Keycloak to {project_name}

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 2]
```diff
@@ -266,6 +266,22 @@ for each section:
     app.get( '/:section/:page', keycloak.protect( protectBySection ), sectionHandler );
 ----
 
+Advanced Login Configuration:
+
+By default, all unauthorized requests will be redirected to the Keycloak login page unless your client is bearer-only. 
+However, a confidential or public client may host both browsable and API endpoints. To prevent redirects on unauthenticated 
+API requests and instead return an HTTP 401, you can override the redirectToLogin function.
+
+For example, this override checks if the url contains /api/ and disables login redirects:
```
Comment (andymunro) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]: Always use URL instead of url

**Review by andymunro — CHANGES_REQUESTED** [review]
Two changes requested.

**Review by andymunro — APPROVED** [review]

**Comment by andymunro** [comment]
@hmlnarik Would you be able to approve this change?

**Comment by andymunro** [comment]
@mposolda Does this look okay to you?

**Review by abstractj — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**